recipe/meta.yaml

```diff
@@ -1,63 +1,89 @@
-{% set version = "3.0.0" %}
+{% set version = "3.4.0" %}
 
 package:
   name: freeglut
   version: {{ version }}
 
 source:
-  fn: freeglut-{{ version }}.tar.bz2
-  url: http://sourceforge.net/projects/freeglut/files/freeglut/{{ version }}/freeglut-{{ version }}.tar.gz
-  md5: 90c3ca4dd9d51cf32276bc5344ec9754
+  url: https://sourceforge.net/projects/freeglut/files/freeglut/{{ version }}/freeglut-{{ version }}.tar.gz
```

Comment:
this is the mirror now, you can also use github, although I'm not sure it matters (maybe github more robust in case they stop mirroring..?)

Reply:
Fixed